src/node/node_to_node.h

```diff
@@ -56,7 +56,9 @@ namespace ccf
 
       if (!recv_authenticated(from, asCb(t), data, size))
       {
-        LOG_INFO_FMT("Dropping invalid authenticated message from {}", from);
+        throw std::logic_error(fmt::format(
```

Comment:
We could always throw a new node-to-node-specific exception here and catch it separately in `raft.h`'s `recv_message()`, and display the "Dropping invalid message from..." message there?

Reply:
Sure, can do. We don't catch any other custom exceptions in `recv_message()` - would it be acceptable to do that there?